mzLib/UsefulProteomicsDatabases/ProteinDbWriter.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Proteomics;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Linq;
+using System.Globalization;
+
+namespace UsefulProteomicsDatabases
+{
+    public class ProteinDbWriter
+    {
+        #region Public Methods
+
+        public static void WriteXmlDatabase(Dictionary<string, HashSet<Tuple<int, ModificationWithMass>>> Mods, List<Protein> proteinList, string outputFileName)
+        {
+            var xmlWriterSettings = new XmlWriterSettings
+            {
+                Indent = true,
+                IndentChars = "  "
+            };
+
+            using (XmlWriter writer = XmlWriter.Create(outputFileName, xmlWriterSettings))
+            {
+                writer.WriteStartDocument();
+                writer.WriteStartElement("uniprot");
```

Comment:
On the other hand, "the first element to appear is called the "root element", which defines what kind of document the file will be. " http://www.tizag.com/xmlTutorial/xmldocument.php

Shouldn't we just call it mzLibProteinDb?